src/screens/MemberDetail/MemberDetail.test.tsx

```diff
@@ -148,28 +148,29 @@ describe('MemberDetail', () => {
     await wait();
 
     userEvent.click(screen.getByText(/Add Admin/i));
+    userEvent.click(screen.getByText(/edit/i));
 
     expect(getByTestId(/dashboardTitleBtn/i)).toBeInTheDocument();
     expect(getByTestId(/dashboardTitleBtn/i)).toHaveTextContent('User Details');
-    expect(screen.getAllByText(/Email/i)).toBeTruthy();
-    expect(screen.getAllByText(/Main/i)).toBeTruthy();
-    expect(screen.getAllByText(/First name/i)).toBeTruthy();
-    expect(screen.getAllByText(/Last name/i)).toBeTruthy();
-    expect(screen.getAllByText(/Member of Organization/i)).toBeTruthy();
-    expect(screen.getAllByText(/Language/i)).toBeTruthy();
-    expect(screen.getAllByText(/Admin approved/i)).toBeTruthy();
-    expect(screen.getAllByText(/Plugin creation allowed/i)).toBeTruthy();
-    expect(screen.getAllByText(/Created on/i)).toBeTruthy();
-    expect(screen.getAllByText(/Admin for organizations/i)).toBeTruthy();
-    expect(screen.getAllByText(/Membership requests/i)).toBeTruthy();
-    expect(screen.getAllByText(/Events/i)).toBeTruthy();
-    expect(screen.getAllByText(/Admin for events/i)).toBeTruthy();
-
-    expect(screen.getAllByText(/Created On/i)).toHaveLength(2);
-    expect(screen.getAllByText(/User Details/i)).toHaveLength(2);
-    expect(screen.getAllByText(/Role/i)).toHaveLength(2);
-    expect(screen.getAllByText(/Created/i)).toHaveLength(4);
-    expect(screen.getAllByText(/Joined/i)).toHaveLength(2);
+    // expect(screen.getAllByText(/Email/i)).toBeTruthy();
```

Comment:
Why were these tests removed?

Reply:
i made this mistake, forgot to uncomment it...fixing it very soon